code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -343,6 +198,7 @@ datum
 			result = "sodiumchloride"
 			required_reagents = list("sodium" = 1, "chlorine" = 1, "water" = 1)
 			result_amount = 3
+			mix_message = "The solution crystallizes with a brief flare of light."
 
 		flash_powder
 			name = "Flash powder"

```

Comment:
Hey so, I forgot to remove these. Could you axe the napalms and the powder recipes so that they don't fuck with the new ones?